types/node/index.d.ts

```diff
@@ -2833,14 +2833,17 @@ declare module "fs" {
      */
     export function readFileSync(filename: string, options?: { flag?: string; }): Buffer;
     export function writeFile(filename: string, data: any, callback?: (err: NodeJS.ErrnoException) => void): void;
+    export function writeFile(filename: string, data: any, encoding: string, callback?: (err: NodeJS.ErrnoException) => void): void;
```

Comment:
Thanks @kimushu. Yes, there are more issues but they are not showstoppers. This PR, as is, should be sufficient to get people going after they upgrade to TS 2.4.1. The other issues can be sorted out later (in separate issues).